metadata/layout.conf

```diff
@@ -1,5 +1,9 @@
 repo-name = defiance
 masters = gentoo
-use-manifests = strict
+use-manifests  = strict
+thin-manifests = true
+manifest-hashes = SHA512 BLAKE2B
+manifest-required-hashes = SHA512
 eapis-banned = 0 1 2 3
 eapis-deprecated = 4
+cache-formats = md5-dict
```

Comment:
Do we want to include this metadata cache in this repo?  It seems easier from a maintenance perspective to keep it out of version control and have consumers generate it as needed.  For an example of another overlay ignoring the cache, see stefantalpalaru/gentoo-overlay@c5a0a6bd.  For an example of another overlay without the cache (listed as stable and official in the layman XML), see https://github.com/FRRouting/gentoo-overlay/.